packages/react-bootstrap-table2/test/bootstrap-table.test.js

```diff
@@ -120,4 +120,109 @@ describe('BootstrapTable', () => {
       expect(body.props().cellEdit.onComplete).toBeDefined();
     });
   });
+
+  describe('handleRowSelect', () => {
+    const rowKey = 1;
+
+    describe('when selectRow.mode is radio', () => {
+      beforeEach(() => {
+        wrapper = shallow(
+          <BootstrapTable
+            keyField="id"
+            columns={ columns }
+            data={ data }
+            selectRow={{ mode: 'radio' }}
+          />
+        );
+      });
+
+      it('state.selectedRowKeys should contain single key', () => {
+        wrapper.instance().handleRowSelect(rowKey);
+
+        expect(wrapper.state('selectedRowKeys')).toEqual([rowKey]);
```

Comment:
You should call `handleRowSelect` again to prove `state.selectedRowKeys` always has one value when single selection.

Reply:
Okey I got it.